### PR TITLE
hide manifest warning toast in prod

### PR DESCRIPTION
**Pull request by archiewood**
### Description

Hides this warning in prod

CleanShot 2024-10-24 at 14 40 10@2x

### Checklist

- [ ] For UI or styling changes, I have added a screenshot or gif showing before & after
- [x] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)
- [ ] I have added to the docs where applicable
- [ ] I have added to the [VS Code extension](https://github.com/evidence-dev/evidence-vscode) where applicable

### CODE REVIEW

File: sites/example-project/src/pages/+layout.js [hunk 1]
```diff
@@ -32,15 +32,17 @@ const loadDB = async () => {
...
-				message: 'Without a manifest file, no data is available'
-			},
-			10000
-		);
+		if (dev) {
+			toasts.add(
+				{
+					id: 'MissingManifest',
+					status: 'warning',
+					title: 'Missing Manifest',
```
Comment (mcrascal) [inline sites/example-project/src/pages/+layout.js:40]:
```suggestion
					title: 'No Sources Found',
```

File: sites/example-project/src/pages/+layout.js [hunk 2]
```diff
@@ -32,15 +32,17 @@ const loadDB = async () => {
...
+					message: 'Without a manifest file, no data is available'
```
Comment (mcrascal) [inline sites/example-project/src/pages/+layout.js:41]:
```suggestion
					message: 'Configure and run sources to include data in your project.'
```

**Review by mcrascal — CHANGES_REQUESTED** [review]
If we're in here, might aswell make it a little friendlier too. AFAIK manifest is not a concept described in the docs.

File: sites/example-project/src/pages/+layout.js [hunk 3]
```diff
@@ -32,15 +32,17 @@ const loadDB = async () => {
 
 	if (Object.keys(renderedFiles ?? {}).length === 0) {
 		console.warn(`Unable to load manifest, do you need to generate sources?`.trim());
```
Comment (archiewood) [inline sites/example-project/src/pages/+layout.js:34]:
```suggestion
		console.warn(`No sources found, do you need to run sources?`.trim());
```

**Review by zachstence — APPROVED** [review]

**Pull request merged**